ReportingServicesTools/Functions/CatalogItems/Rest/Set-RsRestItemDataSources.ps1

```diff
@@ -0,0 +1,105 @@
+# Copyright (c) 2016 Microsoft Corporation. All Rights Reserved.
+# Licensed under the MIT License (MIT)
+
+function Set-RsRestItemDataSources
+{
+    <#
+        .SYNOPSIS
+            This script updates data sources related to a catalog item from the Report Server
+
+        .DESCRIPTION
+            This script updates data sources related to a catalog item from the Report Server
+
+        .PARAMETER RsItem
+            Specify the updated OData payload which was fetched via Get-RsRestItemDataSources.
+
+        .PARAMETER ReportPortalUri
+            Specify the Report Portal URL to your SQL Server Reporting Services Instance.
+
+        .PARAMETER RestApiVersion
+            Specify the version of REST Endpoint to use. Valid values are: "v2.0".
+
+        .PARAMETER Credential
+            Specify the credentials to use when connecting to the Report Server.
+
+        .PARAMETER WebSession
+            Specify the session to be used when making calls to REST Endpoint.
+
+        .EXAMPLE
+            Set-RsRestItemDataSources -RsItem $myReport
```

Comment:
probablyu should add an example of how to get $myReport and how to assign the properties

Reply:
I agree. I was looking at this API and the amount of knowledge required in just setting the data source credentials makes me feel that more work should be put in to make this command usable, if not useful. I am going to pull this and Get-RsItemDataSources out of this PR and check them in later.